jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/unit/TabsUnitTests.java

```diff
@@ -0,0 +1,80 @@
+package io.github.epam.angular.tests.unit;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static io.github.com.StaticSite.angularPage;
+import static io.github.com.pages.AngularPage.tabsSection;
+import static io.github.epam.site.steps.States.shouldBeLoggedIn;
+import static org.testng.Assert.assertTrue;
+
+public class TabsUnitTests extends TestsInit {
+
+    private final List<String> TABS_TITLES_DEFAULT_LIST = Arrays.asList("First", "Second", "Third");
```

Comment:
According to convention, uppercase class field name used with final static elements. It is necessary rename variable or create a constant here.